docker/prometheus/prometheus.yml

```diff
@@ -10,4 +10,6 @@ scrape_configs:
   - job_name: gossamer
     metrics_path: /metrics
     static_configs:
+      # for mac users, change to host.docker.internal:9876 if you want to
```

Comment:
Why do we need this if prometheus is running in the same docker-compose.yml ?

Reply:
You don't need to change if you will run Prometheus + Grafana + Gossamer using docker, but if you're running a Gossamer instance locally (w/o docker) and want to run Prometheus + Grafana using docker you need to change the Prometheus config to find Gossamer using the host. This is the case for me when I was testing Gossamer changes and would like to see the metrics using Promtheus

Reply:
Ok make sense